db/seeders/data/pending-applications.js

```diff
@@ -0,0 +1,449 @@
+'use strict';
+const faker = require('faker');
+const moment = require('moment');
+const uuidv4 = require('uuid/v4');
+
+const projectCountries = ['scotland', 'wales', 'england', 'northern-ireland'];
+const orgTypes = [
+    'unregistered-vco',
+    'unincorporated-registered-charity',
+    'charitable-incorporated-organisation',
+    'not-for-profit-company',
+    'school',
+    'college-or-university',
+    'statutory-body',
+    'faith-group'
+];
+
+const toDateParts = dt => {
+    return { day: dt.date(), month: dt.month() + 1, year: dt.year() };
+};
+
+const mockAddress = () => {
+    return {
+        line1: faker.address.streetAddress(),
+        townCity: faker.address.city(),
+        county: faker.address.county(),
+        postcode: 'B15 1TR'
+    };
+};
+
+module.exports = {
+    data: [
+        {
```

Comment:
Lots of duplication here, is it the expiration date that is useful to change? Could we share a method to avoid duplicating all the shared code?